weaver/core/network/fabric-interop-cc/contracts/interop/go.sum

```diff
@@ -45,14 +45,6 @@ github.com/google/go-cmp v0.6.0 h1:ofyhxvXcZhMsU5ulbFiLKl/XBFqE1GSq7atu8tAmTRI=
 github.com/hashicorp/hcl v1.0.0/go.mod h1:E5yfLk+7swimpb2L/Alb/PJmXilQ/rhwaUYs4T20WEQ=
 github.com/holiman/uint256 v1.2.4 h1:jUc4Nk8fm9jZabQuqr2JzednajVmBpC+oiTiXZJEApU=
 github.com/holiman/uint256 v1.2.4/go.mod h1:EOMSn4q6Nyt9P6efbI3bueV4e1b3dGlUCXeiRV4ng7E=
-github.com/hyperledger/cacti/weaver/common/protos-go/v2 v2.0.0-rc.3 h1:niQG/pUADBRo+o8Aa0C0fQxSnmT+icn6sMQVLPVWSQw=
```

Comment:
I assume these entries got deleted (rather than replaced) because of the import paths' changes? Will we have to wait until the next release to fix this?

Reply:
Yeah I intentionally cleaned these up. When we make the next release, these will be added automatically.